recipes/speclite/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "speclite" %}
+{% set version = "0.7" %}
+{% set sha256 = "2753137e2f3dc0b94d6c3f98c24d674c1a7145410d731cbeb13e0e7f1f7f5ba2" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  sha256: {{sha256}}
+  
+build:
+  number: 0
+  skip: true  # [win]
+  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+
+requirements:
+
+  build:
+    - python x.x
+    - setuptools
+    - astropy
+
+  run:
+    - python x.x
+    - numpy
+    - scipy
+    - astropy
+    - pyyaml
+
+test:
+
+  imports:
+    - speclite
+    
+  commands:
+    # Execute the full test suite
+    
+    - cd ${PREFIX}/lib/python*/site-packages/speclite/tests ; py.test -vv
+  
+  requires:
+    - pytest
+    - pyyaml
+    - matplotlib
+
+about:
+  home: http://speclite.readthedocs.io/en/latest/overview.html
+  license: BSD-3-Clause
+  license_family: BSD
+  summary: 'This package provides a set of lightweight utilities for working with spectroscopic data in astronomy. Based on the astropy affiliated package template.'
+  license_file: '{{ environ["SRC_DIR"] }}/licenses/LICENSE.rst'
```

Comment:
I gather the license file is in a sub-directory of the source distribution so you should be able to replace this with just `licenses/LICENSE.rst`.

Reply:
Though it isn't required, adding this to `about` would be helpful:

```
    dev_url: https://github.com/dkirkby/speclite
```